### DISCUSSION TITLE
Sprint 0 Review y Sprint 1 Planning 23/04

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [NataliaAlvarezIspc/proyecto-ispc-ecommerce#155] [General]
Apuntes de la reunión: 

**PLANNING:**

- Estuvimos revisando los requisitos a cumplir para el Sprint 1, para el  cual tenemos como fecha límite el 05/05:

     _"Convertir los archivos .html del proyecto previo, en una SPA en Angular con módulos (Layouts/Shared y Pages son un ejemplo) 
     con sus components correspondientes (HOME, REGISTRO, LOGIN, DASHBOARD).
     Crear módulos y componentes para la tienda virtual o ecommerce (PRODUCTO o SERVICIOS, PRODUCTO INDIVIDUAL, COMPRA).
     Incorporar la navegabilidad de la aplicación mediante Routing con Angular."_

En cuanto a la parte de backend, para este sprint, ya la tenemos realizada aunque puede ser que haya que hacerle algunas modificaciones.

- Realizamos conjuntamente el armado inicial del proyecto en Angular con los módulos Shared y Pages y los componentes iniciales, luego cada uno irá sumando los componentes necesarios, de acuerdo a las tareas que vayan realizando. Esta primera maquetación está subida al repositorio.

- Forma de trabajo: cada uno se asignará las tareas que realizará para este sprint, las cuales ya están creadas y se encuentran en el backlog. Luego las subirá a la rama correspondiente de cada tarea y solicitará el pull request para ser mergeado al dev. Las tareas para este sprint son todas las de maquetación y front (no las de back), para las tareas de maquetación, solo se subirá la imagen a la issue correspondiente y se dará por cerrada. Es importante ir acomodando las issue en el tablero Kanban a medida que vayamos trabajando para tener una idea en que situación estamos.

- A mediados de esta semana haremos una evaluación para de la situación en la que nos encontramos para ver si tenemos que hacer ajustes.

- Vimos la posibilidad de realizar nuestra reunión semanal otro día para que todos podamos conectarnos. Un día y horario tentativo podría ser los sábados a las 22 hs. Esto lo iremos consensuando en la semana.

**REVIEW:**

- Completamos todos los requerimiento para el Sprint 0. Incluso adelantamos algunos (backend) del Sprint 1 y el Sprint 2.
- Tenemos creadas las tareas para el Sprint 1.
- Terminamos con todas las tareas antes de la fecha solicitada (documentación, presentación del grupo y del proyecto, diagramas).
- Tuvimos dos bajas en el medio del sprint, cuando el proyecto y el grupo ya estaban presentados. Además algunos miembros tuvieron poca participación. Esto produjo una sobrecarga de tareas sobre pocos participantes.

**Comment by rpgrca** [comment]
Sprint 1 Planning

**Reply by NataliaAlvarezIspc** [reply]
Corregido. Gracias!

**Comment by BraianTroncoso** [comment]
:+1:
